inc/functions.php

```diff
@@ -1906,28 +1906,295 @@ function my_set_array_cookie($name, $id, $value, $expires="")
 	$mybb->cookies['mybb'][$name] = $newcookie;
 }
 
+/*
+ * Arbitrary limits for _safe_unserialize()
+ */
+define('MAX_SERIALIZED_INPUT_LENGTH', 4096);
+define('MAX_SERIALIZED_ARRAY_LENGTH', 256);
+define('MAX_SERIALIZED_ARRAY_DEPTH', 5);
```

Comment:
@Sama34 , `my_unserialize()` is also used on scripts which don't include `inc/init.php`, for example installation. I think they're safer to use here or in the `_safe_unserialize()` function body.

But are these limits required for fixing the issue anyways?